### PR TITLE
Setup/style

### PR DESCRIPTION
**Pull request by M-AminAlizadeh**
The things that I did in this PR:
- Add style using vanilla CSS precisely like the design
- Fix the reload requirement to display book immediately after user clicks on the add book button

### CODE REVIEW

File: src/Components/form.module.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.formContainer {
```
Comment (shahadat3669) [inline src/Components/form.module.css:1]: - [ ] The spacing between the `formContainer` and `books-page` sections isn't consistent with the design. To ensure better alignment, please adjust the spacing to match the design's specifications. This attention to detail will enhance the overall visual coherence of your project. Keep up the good work! 🛠️📐🎨

| Your | Design |
| --- | --- |
|  image| image|

File: src/Components/Form.jsx [hunk 2]
```diff
@@ -1,44 +1,62 @@
...
-            setTitle(e.target.value);
-          }} 
-          />
-      <input type="text" placeholder="Book author" value={author}
-          onChange={(e) => setAuthor(e.target.value)} />
-      
-      <button type="submit" onClick={handleClickAddBtn}>ADD BOOK</button>
-    </form>
+    <>
+      <form className={styles.formContainer}>
```
Comment (shahadat3669) [inline src/Components/Form.jsx:38]: - [ ] Kindly realign the layout to adhere to the design specifications for this section. Presently, the button's placement isn't aligned as intended. By ensuring that you closely follow the design's layout guidelines and adjust the positioning of the button and the gap between elements accordingly, you will achieve a more polished and visually appealing result. Your attention to these finer points will contribute to a seamless user experience. Keep up the great work! 📏🎨👍

image

File: src/style.css
Comment (shahadat3669) [inline src/style.css:1]: - [ ] Well, there's a minor adjustment needed to match the design's color scheme. Please modify the background color of all sections bellow the header to align with the design's specifications. This subtle yet important change will ensure that your project remains consistent with the intended visual aesthetic. Keep up the good work! 🎨🖌️👍

| Your | Design |
| --- | --- |
|  image| image|

File: src/Components/Books.jsx [hunk 3]
```diff
@@ -17,7 +18,7 @@ const Books = ()=> {
   }
 
   return (
-  <div className="books">
+  <div className={styles.books}>
       {Object.entries(books).map(([key, bookItems]) => bookItems.map((book) => (
```
Comment (shahadat3669) [inline src/Components/Books.jsx:22]: - [ ] [Optional] Well, you've done a great job implementing the book addition feature. However, for a more seamless user experience in line with the project requirements, consider implementing an automatic update of the book details without requiring a manual page refresh upon adding a new book. This enhancement will contribute to the overall fluidity of your application and provide a more intuitive interaction for users. Keep up the great work! 📚🔄👏

[Bookstore.webm](https://github.com/M-AminAlizadeh/Bookstore/assets/55840999/b77c1bee-cd8d-42c0-93a8-bdaf1622744f)

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @M-AminAlizadeh , 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ Good use of modular CSS. 👍 
✅ The code is clean and organized! 💯


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Form.jsx [hunk 4]
```diff
@@ -1,44 +1,62 @@
...
+            placeholder="Book title" 
+            value={title}
+            onChange={(e) => {
+              setTitle(e.target.value);
+            }} 
+            className={`${styles.input} ${styles.inputBookTitle}`}
+            />
+
+        <input type="text" placeholder="Book author" value={author}
+            onChange={(e) => setAuthor(e.target.value)} className={`${styles.input} ${styles.inputBookAuthor}`} required/>
```
Comment (topeogunleye) [inline src/Components/Form.jsx:50]: - [ ] I noticed that the changes you made may have made it not possible for me to see how your Added books looks like as I can't add a book successfully. See video below for more explanation:

https://www.loom.com/share/0c98c849943346bfb5f1190d53b28b7b?sid=4b391cdc-8e01-4c69-9cf4-a8c27109642c

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @M-AminAlizadeh, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* The width and height of the profile container are equal. ✔️ 
* Applied a `border-radius` of 4px to impart a pleasing curvature. ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Form.jsx (hunk 4 above)
Comment (M-AminAlizadeh) [reply src/Components/Form.jsx:50]: @topeogunleye Thank you for your issue reporting
 I will fix it

**Review by vickymarz — APPROVED** [review]
Hi @M-AminAlizadeh, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**